src/modules/change-state.js

```diff
@@ -279,5 +283,7 @@
 	App.modules.exports('change-state', {
```

Comment:
I think we should get rid of the module instead keeping it for the sake of legacy.

We should put change-state.js in a new `fx` folder too 😁

Reply:
@PascalPiche wanted to have a simple upgrade path, and I agreed. It will get moved soon enough.